iOS/Evol.Me/Podfile

```diff
@@ -17,7 +17,6 @@ pod 'AFNetworking', '~> 2.5'
 use_frameworks!
 pod 'performSelector-swift'
 pod 'UIColor+FlatColors'
```

Comment:
**Comment:**
I agree. This sounds good. Also, please remove all changes.

**Rationale:**
We need to remove all unnecessary frameworks.

**Reference:**
https://dev.mysql.com/doc/internals/en/coding-style.html